satpy/readers/fci_l2_nc.py

```diff
@@ -469,6 +472,21 @@ def nc(self):
             }
         )
 
+    @property
+    def wind_time(self):
+        """Return wind time as a datetime object.
+
+        This quantity is already present as a wind_time dataset,
+        in units of Seconds since 2000-01-01 00:00:00.0.
```

Comment:
Maybe also mention what the time represents, like this:

```
This quantity represents the `start_time` of the second image used for the AMV tracking and is already present as a wind_time dataset, in units of seconds since 2000-01-01 00:00:00.0.
```

Reply:
So this would be the `nominal_time` of the latter FCI image used in the retrieval, not the "actual" sensing time of the location? I assume the retrieval is done as a continuous field with a optical flow approach or such, so that it would be hard to say what's the exact sensing time for each retrieval. So, if I got this right, should the naming be `nominal_wind_time`?